pygeoapi/api.py

```diff
@@ -45,11 +48,13 @@
 TEMPLATES = '{}{}templates'.format(os.path.dirname(
     os.path.realpath(__file__)), os.sep)
 
+#: Return headers for requests (e.g:X-Powered-By)
```

Comment:
is `#:` intended (as opposed to `#`) ?

Reply:
The `:#` is necessary to indicate RTD that it should that string to describe the content below in this case the headers (it needs to stay)